client/my-sites/current-site/sidebar-banner/style.scss

```diff
@@ -1,9 +1,13 @@
-.sidebar-banner {
+a.sidebar-banner {
 	display: flex;
 	background-color: $alert-green;
 	color: $white;
 	font-size: 12px;
 
+	:visited {
+		color: $white;
```

Comment:
Thanks for this, I noticed there was something odd but I couldn't pinpoint it exactly :)